src/set_theory/game/pgame.lean

```diff
@@ -148,6 +148,28 @@ def move_right : Π (g : pgame), right_moves g → pgame
 @[simp] lemma right_moves_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).right_moves = xr := rfl
 @[simp] lemma move_right_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).move_right = xR := rfl
 
+/-- Conway induction on games. -/
+@[elab_as_eliminator] def move_rec_on {C : pgame → Sort} (x : pgame)
+  (IH : ∀ (y : pgame), (∀ i, C (y.move_left i)) → (∀ j, C (y.move_right j)) → C y) : C x :=
+begin
+  cases x with xl xr xL xR,
+  apply pgame.rec_on,
+  intros yl yr yL yR IHl IHr,
+  apply IH, exact IHl, exact IHr
+end
+
+/-- `is_option x y` means that `x` is either a left or right option for `y`. -/
+def is_option (x y : pgame) : Prop :=
+(∃ i, y.move_left i = x) ∨ ∃ i, y.move_right i = x
```

Comment:
This might be easier to work with as an inductive,
```suggestion
inductive is_option : pgame → pgame → Prop
| left {x} {i} : is_option (y.move_left i) y
| right {x} {j} : is_option (y.move_right j) y
```
as this gives you a convenient recursor

Reply:
You can get back your definition from mine with `@[mk_iff]`